pinot-common/src/main/java/org/apache/pinot/common/swagger/SwaggerSetupUtils.java

```diff
@@ -50,12 +50,6 @@ public static void setupSwagger(String componentType, String resourcePackage, bo
     beanConfig.setResourcePackage(resourcePackage);
     beanConfig.setScan(true);
 
-    try {
```

Comment:
Do we want to set the host based on the config for better customization? We can make a separate issue to support that.

Reply:
It was introduced in #5941, and doesn't seem required. We can remove it for now and revisit later. This seems like a bug to me